src/pyhf/infer/test_statistics.py

```diff
@@ -82,7 +84,8 @@ def qmu(mu, data, pdf, init_pars, par_bounds):
         >>> init_pars = model.config.suggested_init()
         >>> par_bounds = model.config.suggested_bounds()
         >>> par_bounds[model.config.poi_index] = [-10.0, 10.0]
-        >>> pyhf.infer.test_statistics.qmu(test_mu, data, model, init_pars, par_bounds)
+        >>> fixed_vals = []
+        >>> pyhf.infer.test_statistics.qmu(test_mu, data, model, init_pars, par_bounds, [])
```

Comment:
```suggestion
        >>> pyhf.infer.test_statistics.qmu(test_mu, data, model, init_pars, par_bounds, fixed_vals)
```

As `fixed_vals=[ ]` is used above, we should use the `fixed_vals` variable here.